projects/plugins/jetpack/_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -824,7 +824,7 @@ public static function get_openai_jwt() {
 				'method'  => 'POST',
 				'headers' => array( 'Content-Type' => 'application/json; charset=utf-8' ),
 			),
-			wp_json_encode( array() ),
+			wp_json_encode( array(), JSON_UNESCAPED_SLASHES ),
```

Comment:
Since we're sending a content type with `charset=utf-8`, should be safe to `JSON_UNESCAPED_UNICODE` here.